### PR TITLE
Make getContentsDescription usable with a Stream [#56]

### PR DESCRIPTION
**Pull request by RobbWatershed**
Make getContentsDescription usable with a Stream

### CODE REVIEW

**Comment by gotson** [comment]
I fixed the issue with Travis not showing the build. Build is failing, can you make sure checkstyle rules are followed?

**Comment by RobbWatershed** [comment]
Thanks for fixing Travis, that's much clearer like that. Fix posted !

File: src/main/java/com/github/junrar/Junrar.java [hunk 1]
```diff
@@ -90,6 +90,24 @@
...
+                logger.warn("archive is encrypted cannot extract");
+                return new ArrayList<>();
+            }
+            for (final FileHeader fileHeader : arch) {
+                contents.add(new ContentDescription(fileHeader.getFileName(), fileHeader.getUnpSize()));
+            }
+        } finally {
+            arch.close();
+        }
+        return contents;
```
Comment (gotson) [inline src/main/java/com/github/junrar/Junrar.java:108]: You can probably pull that duplicate code out in a private function

Comment (RobbWatershed) [reply src/main/java/com/github/junrar/Junrar.java]: Absolutely !

**Comment by RobbWatershed** [comment]
Awesome, thanks !

**Pull request merged**